### ISSUE TITLE
@Schema annotation not taken into account for data description

### ISSUE THREAD

**Issue by ysavourel** [javalin/javalin-openapi#188]
Hi,
I am trying to migrate to the new OpenAPI plugin (5.5.0-1) and I cannot manage to get the @Schema annotations (of io.swagger.v3.oas.annotations.media) to be taken into account by the plugin.
I have noticed the @OpenApiDescription and @OpenApiRequired (of io.javalin.openapi) and those seem to be processed properly. But they are not enough to describe the properties of the DTOs. For example defaultValue, nullable, maxLength, etc. that exist for @Schema are not available.

I do have the openapi-annotation-processor set in my POM and it seems to work properly. Do I need another processor for the @Schema annotations?
In short: How can I re-use @Schema or use other annotations with the latest version of the OpenApi plugin for Javalin to generate a properly documented schema in the Swagger page?

Thanks
-yves

**Comment by dzikoysk** [comment]
Hi, @Schema from Swagger is not supported. There's a full response in similar issue with a reference to the solution:

* https://github.com/javalin/javalin-openapi/issues/134#issuecomment-1366917071

**Comment by ysavourel** [comment]
Thanks for the pointer.
A mix of @Custom and defining a few @CustomAnnotation seems to resolve the problem.
